vendor/engines/bulk_email/config/locales/en.yml

```diff
@@ -11,4 +11,20 @@ en:
       start: Starting
       end: Ending
     no_results: No users found
-    export_html: Export
+    compose_mail: Compose Mail
+    export: Export
+    body:
+      greeting: Hello %{recipient_name},
+      signoff: If you have any questions, please contact %{facility_name}.
```

Comment:
Linking to the facility is a good idea. I'd like to get feedback from NU on how they'd like this to read.